install_themes/action.yml

```diff
@@ -54,3 +54,55 @@ runs:
             ${{ inputs.container }} \
             composer update ${{ inputs.update_options }}
         fi
+
+    - name: Generate deprecation warning to summary page
+      shell: bash
+      run: |
+        echo "Deprecation Warning:" >> $GITHUB_STEP_SUMMARY
+          echo "---------------------" >> $GITHUB_STEP_SUMMARY
+          echo -e "\033[0;31m### This actions are deprecated and may not be supported ${{ inputs.install_theme}} \033[0m" >> $GITHUB_STEP_SUMMARY
+          exit 1
+
+    - name: 'Output report'
+      shell: bash
+      run: |
+        for FILE in $(find reports -type f|sort); do
+          cat "${FILE}" >>github_report.md
+        done
+        cat github_report.md >>"${GITHUB_STEP_SUMMARY}"
+        
+        for FILE in $(find slack_report -type f|sort); do
+          cat "${FILE}" >>slack_report.md
+        done
+        SLACK_MARKDOWN=$(cat slack_report.md|sed 's/$/\\n/g' | tr -d '\n')
+        cat >slack_deprecated.json <<EOF
```

Comment:
This json looks like an example from the slack builder. it does not have a header nor a text indicating that "install_themes" was called or by which workflow it was called. Instead, it contains a dummy text and a button.

Look at the message preview on the left of the [slack builder using your json]( https://app.slack.com/block-kit-builder/T03LW1LM5#%7B%22blocks%22:%5B%7B%22type%22:%22section%22,%22text%22:%7B%22type%22:%22mrkdwn%22,%22text%22:%22This%20is%20a%20section%20block%20with%20a%20button.%22%7D,%22accessory%22:%7B%22type%22:%22button%22,%22text%22:%7B%22type%22:%22plain_text%22,%22text%22:%22Click%20Me%22,%22emoji%22:true%7D,%22value%22:%22slack_webhook_notify_devops%22,%22url%22:%22https://hooks.slack.com/services/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%22,%22action_id%22:%22button-action%22%7D%7D%5D%7D) and tell me, how you would figure out that it is a deprecation warning and where it is being used.

It also makes sense to tell people about where the action was used as we need to know where to remove it from. Looking at [call-universal_test:workflow yml](https://github.com/OXID-eSales/github-actions/blob/main/.github/workflows/call-universal_test_workflow.yml#L116), you can see, how I assembled a link to the job there. You can do something similar and put it into the slack message as well.